From 7a6937966aee94fe2dfca77a2f74a41767a03fa4 Mon Sep 17 00:00:00 2001
From: Andreas Oberritter <obi@opendreambox.org>
Date: Wed, 6 Feb 2013 20:54:59 +0100
Subject: [PATCH 1/2] Fix compile with kernel 3.2

include/rtl871x_cmd.h:107:25: error: field 'event_tasklet' has incomplete type
include/rtl871x_recv.h:205:24: error: field 'recv_tasklet' has incomplete type
include/rtl871x_xmit.h:355:24: error: field 'xmit_tasklet' has incomplete type
os_intf/linux/usb_intf.c:1028:9: error: 'THIS_MODULE' undeclared
---
 include/osdep_service.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/osdep_service.h b/include/osdep_service.h
index e72304a..7f5fad7 100755
--- a/include/osdep_service.h
+++ b/include/osdep_service.h
@@ -51,6 +51,8 @@
 	#include <linux/etherdevice.h>
 	#include <net/iw_handler.h>
 	#include <linux/proc_fs.h>	// Necessary because we use the proc fs
+	#include <linux/interrupt.h>
+	#include <linux/module.h>
 	
 #ifdef CONFIG_USB_HCI
 	typedef struct urb *  PURB;
-- 
1.7.10.4

